benchmarks/linear_programming/utils/benchmark_lp_mittelmann.sh

```diff
@@ -30,12 +30,12 @@ echo "Download done"
 # EAGER module loading to simulate real-life condition
 export CUDA_MODULE_LOADING=EAGER
 
-# Benchmark all instances (cuOpt needs to be compiled first)
+# Benchmark all instances (cuOpt needs to be compiled first, you can compile in LP only mode and you should turn on BUILD_LP_BENCHMARKS)
 for instance in ${CUOPT_HOME}/benchmarks/linear_programming/datasets/*/ ; do
     # Will generate the solver log for each instance. Could addtionally generate the solution file by uncommenting the --solution-path
     instance_name=$(basename $instance)
     echo "Parsing ${instance_name}.mps then solving"
-    ${CUOPT_HOME}/cpp/build/solve_MPS_file --path ${CUOPT_HOME}/benchmarks/linear_programming/datasets/${instance_name}/${instance_name}.mps --time-limit 3600 # --solution-path $CUOPT_HOME/benchmarks/linear_programming/datasets/$instance.sol
+    ${CUOPT_HOME}/cpp/build/solve_LP --path ${CUOPT_HOME}/benchmarks/linear_programming/datasets/${instance_name}/${instance_name}.mps --time-limit 3600 # --solution-path $CUOPT_HOME/benchmarks/linear_programming/datasets/$instance.sol
```

Comment:
Let's start using cuopt_cli here?  That's what Hans would be using as well.

Reply:
same comment as below